### ISSUE TITLE
E2E tests which check installation of Eclipse Che from OperatorHub failed on OCP 4.4

### ISSUE THREAD

**Issue by dmytro-ndp** [eclipse-che/che#16253]
### Describe the bug
Command to run test tests:

> docker run --shm-size=4096m -p 5920:5920 -e TS_SELENIUM_HEADLESS=true -e TS_SELENIUM_LOAD_PAGE_TIMEOUT=420000 -e TS_SELENIUM_WORKSPACE_STATUS_POLLING=20000 -e TS_SELENIUM_BASE_URL=***** -e TS_SELENIUM_LOG_LEVEL=DEBUG -e TS_SELENIUM_USERNAME=admin -e TS_SELENIUM_PASSWORD=admin -e TS_SELENIUM_MULTIUSER=true -e DELETE_WORKSPACE_ON_FAILED_TEST=true -e TS_SELENIUM_START_WORKSPACE_TIMEOUT=900000 -v /mnt/hudson_workspace/workspace/crw-typescript-tests/test-repo/tests/e2e:/tmp/e2e:Z -e TEST_SUITE=test-operatorhub-installation -e TS_SELENIUM_HAPPY_PATH_WORKSPACE_NAME=test-operatorhub-installation -e TS_SELENIUM_WEB_CONSOLE_OCP_URL=****** -e TS_SELENIUM_CATALOG_TILE_OPERATOR_NAME=crw-preview -e 'TS_SELENIUM_OPERATOR_LOGO_NAME=Red Hat CodeReady Workspaces' -e TS_SELENIUM_INSTALL_PROJECT_NAME=crw-operatorhub-tls-oauth -e TS_OCP_OPERATOR_UPDATE_CHANNEL=final -e TS_SELENIUM_VALUE_TLS_SUPPORT=true -e TS_SELENIUM_VALUE_SELF_SIGN_CERT=true -e TS_SELENIUM_VALUE_OPENSHIFT_OAUTH=true -e TS_SELENIUM_INSTALL_APP_PREFIX_URL=codeready -e TS_OCP_LOGIN_PAGE_PROVIDER_TITLE=htpasswd -e TS_SELENIUM_OCP_USERNAME=***** -e 'TS_SELENIUM_OCP_PASSWORD=******' -e TS_SELENIUM_W3C_CHROME_OPTION=false eclipse/che-e2e:nightly

Error log:
```
        ▼ OcpLoginPage.waitOpenShiftLoginPage
      1) Open login page


  0 passing (23s)
  1 failing

  1) E2E
       Go to OCP and wait console OpenShift
         Open login page:
```

### Screenshots
![screenshot-Open_login_page](https://user-images.githubusercontent.com/1197777/76072971-8b51f900-5fa1-11ea-96e6-e8f5c0254a28.png)

**Comment by artaleks9** [comment]
The reason of it is new widget of _Login Page_ on OCP 4.4. Look at e.g. on _Login Page_ OCP 4.3:
![ocp43-login-page](https://user-images.githubusercontent.com/5408906/76091113-9b7bcf80-5fc5-11ea-9446-4e327b62f04b.png)

So I'm going to update locator in a method which uses it and modify some steps that it would work on both OCP 4.3 and OCP 4.4.

**Comment by artaleks9** [comment]
Resolved by merge PR #16279.
